src/app/api/utils/auth.ts

```diff
@@ -222,3 +222,8 @@ export function bearerToken(req: NextRequest) {
   // The remaining portion, which should be the token
   return authHeader.substring(authHeader.indexOf(" ") + 1);
 }
+
+export function isAdmin(email: string) {
+  const admins = AppConstants.ADMINS?.split(",") ?? [];
```

Comment:
Is `,` allowed in email addresses? I think it might technically be... since this is set in the env var and we control that I think it's OK in this case, just something to be aware of.

Reply:
hmm interesting.. We might need another way to separate the addresses